fn main() {
    // unrecoverable error
    panic!("Crashed");
}